Language/Functions/Bits and Bytes/bitSet.adoc

```diff
@@ -46,6 +65,7 @@ Nothing
 
 [float]
 === See also
+bitClear()
```

Comment:
```suggestion

[role="language"]
* #LANGUAGE# link:../bitclear[bitClear()]
```

This is required to make it a link:

https://github.com/arduino/reference-en/blob/074f4d2f081c9d6f7251b08b990b09709c3c580b/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc?plain=1#L111-L119